packages/fuel-indexer-tests/contracts/fuel-indexer-test/Forc.lock

```diff
@@ -1,13 +1,13 @@
 [[package]]
-name = 'core'
-source = 'path+from-root-63B5D95B29B128A3'
+name = "core"
```

Comment:
@lostman 
- So if you want to upgrade Sway to 0.46 is this PR that's fine with me - I believe beta-4 is on 0.46 now anyway
- But let's make sure to upgrade _all_ smart contracts to 0.46 in a single go (for the ones that aren't on 0.46 already, assuming there are any)